src/styles/partials/components/_explore.scss

```diff
@@ -0,0 +1,56 @@
+.dg_explore-link {
+  background: $gray-lightest;
+  display: block;
+  font-family: $default-heading-font;
+  font-weight: 600;
+  font-size: $smallest-heading-font-size;
+  min-height: 130px;
+  margin-bottom: $default-margin;
```

Comment:
do we have variables for font weight and height?

Reply:
yes, will update.

Reply:
height, this is a one off, i will add a comment.

Reply:
added comment, resolving.